mne/io/egi/tests/test_egi.py

```diff
@@ -475,3 +475,14 @@ def test_meas_date(fname, timestamp, utc_offset):
     assert raw.info['meas_date'] == measdate
     assert raw.info['utc_offset'] == utc_offset
     assert local_utc_diff == int(utc_offset[:-2])
+
+
+@requires_testing_data
+@pytest.mark.parametrize('fname, standard_montage', [
+    (egi_mff_fname, 'GSN-HydroCel-129'),  # 129 chan EGI file
+    (egi_mff_pns_fname, 'GSN-HydroCel-257')  # 257 chan EGI file
+])
+def test_set_standard_montage(fname, standard_montage):
+    """Test setting a standard montage."""
+    raw = read_raw_egi(fname, verbose='warning')
+    raw.set_montage(standard_montage, match_alias=True)
```

Comment:
Is there anything in `raw.ch_names`, `raw.info['dig']`, `raw.info['chs'][ii][...]` that are affected by this PR? Or it's just that this `set_montage` no longer raises an error?

Reply:
I *think* that it will change, because after https://github.com/mne-tools/mne-python/pull/10898 the ``VREF`` channel is still stored as a proper (flat) channel?  So its `dig` entry will update when setting montage.

Reply:
If that's the case then can you update this test to show that something has changed with these variables? Like assert that some channel name is in `.ch_names` that didn't used to be there, or there is some new dig entry, etc.?

Reply:
> Is there anything in `raw.ch_names`, `raw.info['dig']`, `raw.info['chs'][ii][...]` that are affected by this PR? Or it's just that this `set_montage` no longer raises an error?

No, this PR should only affect `set_montage`. I added this test in case the standard montage was to one day change in a way that affects this alias mapping, this test would catch it?

Reply:
Okay then we can just keep this as is, then!

Reply:
Argh sorry I just saw  the comment by @drammock ! I can try to check this.

Reply:
> I _think_ that it will change, because after #10898 the `VREF` channel is still stored as a proper (flat) channel? So its `dig` entry will update when setting montage.

@larsoner , @drammock is correct, the `dig` entry for `VREF` does update when setting the montage, as do the dig entries for all the electrodes (which is intended).

```
from pathlib import Path
from mne.datasets.testing import data_path
import mne

testing_path = data_path()
egi_path = testing_path / 'EGI'
egi_fname = egi_path / 'test_egi.mff'

raw = mne.io.read_raw_egi(egi_fname)
print('before montage: ', raw.info['dig'][-1])  # dig info for VREF before setting montage
# EEG #129 : (0.0, 13.2, 138.1) mm

raw.set_montage('GSN-HydroCel-129', match_alias=True)
print('after montage: ', raw.info['dig'][-1])  # dig info for VREF after setting montage
# EEG #129 : (0.0, 12.5, 131.0) mm
```

Reply:
I think an easy test would be to check the length of `raw.info['dig']` after `set_montage(match_alias=True, on_missing='ignore')`.  On `main` the length should be 131, since VREF will be ignored when setting the montage. On this PR it ought to be 132 though (128 eeg + 1 ref + 3 cardinal).

Reply:
Okay that's a good idea, updating the test and will push.